physical/cockroachdb/cockroachdb.go

```diff
@@ -51,6 +56,11 @@ func NewCockroachDBBackend(conf map[string]string, logger log.Logger) (physical.
 		return nil, fmt.Errorf("missing connection_url")
 	}
 
+	var haEnabled bool
+	if haEnabledStr, ok := conf["ha_enabled"]; ok && haEnabledStr == "true" {
```

Comment:
This is fine as is, but since in Go an absent key looked up in a map results in the zero value, and this is a map[string]string, you can just say `haEnabled = conf["ha_enabled"] == "true"` if you want to be a bit more succinct.